aip/general/0193.md

```diff
@@ -138,6 +138,18 @@ descriptions, and examples:
     `ErrorInfo`,`LocalizedMessage`, and so on.
   - *Example*: Described in the [ErrorDetails](#error-details) section.
 
+
+**Important:** In the context of the of the `google.rpc.Status` Protobuf
+message, the value of the field `code` is the numeric equivalent of the
+enum value chosen from [`google.rpc.Code`][Code]. Meanwhile, when the
+error is exressed as JSON, e.g. in the [sample
+above](#json-representation), the value of the field by the same name,
+"code", is the HTTP status code equivalent of the selected
+`google.rpc.Code`. For example, if [`INVALID_ARGUMENT`][InvalidArgument]
```

Comment:
Something must be unclear about this sentence. "Conversely" is appropriate here as this is a specific example of the application of this distinction--between `code` in proto and "code" in JSON. Can you recommend a clarifying edit, @jschneider11?

Reply:
It might be better to put the example right after each description. Like this:

**Important:** In the context of the of the `google.rpc.Status` Protobuf
message, the value of the field `code` is the numeric equivalent of the
enum value chosen from [`google.rpc.Code`][Code]. For example, if
[`INVALID_ARGUMENT`][InvalidArgument] is chosen, the value of `code` (in
the context of the Protobuf message, `google.rpc.Status`) will be `3`.
However, when the error is expressed as JSON, as in the [sample above](#json-representation), the
value of the field by the same name, "code", is the HTTP status code
equivalent of the selected `google.rpc.Code`. For example,  the value of the field "code" in the JSON reply will be `400`.

Reply:
Fixed.

Reply:
Looks good.